File: nimbus/View/Components/TableCells/ObjC/TableCellClass.m
//
//  TableCellClass.m
//  nimbus
//
//  Created by Pericles Maravelakis on 02/07/2018.
//	periclesm@cloudfields.net
//	Licensed under Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
//	https://creativecommons.org/licenses/by-sa/4.0/
//

#import "TableCellClass.h"

@implementation cloudCell
@synthesize cloudInitials, cloudName, cloudDetail;

- (void)awakeFromNib {
	[super awakeFromNib];
	self.backgroundColor = [UIColor clearColor];
}

@end

File: nimbus/View/Components/TableCells/TableCellClass.swift
//
//  TableCellClass.swift
//  nimbus
//
//  Created by Pericles Maravelakis on 1/2/20.
//  Copyright © 2020 Cloudfields. All rights reserved.
//

import UIKit

class cloudCell: UITableViewCell {
	
    @IBOutlet weak var cloudInitials: UILabel!
    @IBOutlet weak var cloudName: UILabel!
    @IBOutlet weak var cloudDetail: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        self.backgroundColor = UIColor.clear
    }
}
